### ISSUE TITLE
SC 3.3.8 Re-entering usually isn't essential

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag#1346]
The definition of "essential" does not apply to things like re-entering an e-mail address or password. Removing a re-entry field does not "fundamentally change the information or functionality of the content". I do not think adding a note that says otherwise addresses this. Notes are not normative. They can not be used to "modify" normative requirements. I think a different phrase is needed for this SC.

**Comment by WilcoFiers** [comment]
Would be addressed by adopting my suggestion in #1431.

**Comment by mbgower** [comment]
> Removing a re-entry field does not "fundamentally change the information or functionality of the content". 

I'm not sure I agree. Scenario: you do not prompt the user to re-enter their password. Although they intended to type "password" they typed "passsword" (three Ss). You have no way of knowing their intent and accept their typed password. They cannot get into the system with the password they believe they created, so the functionality of the content (creating a user name and password, or resetting a password) has been fundamentally changed.

Password re-entry developed as a practice because it clearly reduced unintentional user error. It is almost globally adopted, which speaks to its essential nature.

**Comment by alastc** [comment]
The group agreed to leave the SC text as it is today: https://www.w3.org/2021/02/23-ag-minutes.html#item07

I'll note key factors were that: 
- We didn't want to bake-in a particular solution that might change in future.
- There is pretty universal agreement that having a second entry is essential for it's purpose. 
-  Given that, it appears to meet the definition: "if removed, would fundamentally change the information or functionality of the content, and information and functionality cannot be achieved in another way that would conform"
- There was some discussion about 1.3.5, but we've covered that before in #1080 

I was going to suggest an update to the understanding document, but we already have:

- A note underneath the SC text: "Security verification, such as repeating a password, is considered essential."
- A line in the intent: "There is an exception for essential uses of input re-entry for things like password entry (security), or memory games which would not be possible if the previous answers were supplied."

I believe @WilcoFiers said he could live with SC text, can we close this?
